drivers/include/at86rf2xx.h

```diff
@@ -251,6 +251,7 @@ typedef struct {
     uint16_t flags;                         /**< Device specific flags */
     uint8_t state;                          /**< current state of the radio */
     uint8_t tx_frame_len;                   /**< length of the current TX frame */
+    uint8_t status;                         /**< Tx/Rx status (TRAC_STATUS) */
```

Comment:
Why is this value kept in the device descriptor?
The TRAC_STATUS only reports the status of TX with automatic retransmissions. This can be read directly on TX_DONE (as it was before)

Reply:
In `confirm_send()` the status of the last transmission is evaluated, and in the `_isr()` the status is saved.
If TRAC_STATUS was not needed in `_isr()`, then I guess I could also read the register in `confirm_send()` and not save it.
But looking at the code of `openthread_netdev.c :: _event_cb()`, the event is passed to `send_pkt()` which distinguishes between cca failure, no ACK and no error. So that´s why I kept the `#ifdef MODULE_OPENTHREAD` block inside `_isr()`.

Reply:
We could do the same in `_isr` when `#ifdef MODULE_OPENTHREAD`. As mentioned earlier, this status only tells what was the result of TX_ARET (it's not even read in BASIC_MODE).
 
Besides that, we could drop these lines if we do a minor adoption to the event callbacks of netdev. At least GNRC is able to differentiate TX_COMPLETE from TX_COMPLETE_DATA_PENDING.

Reply:
Note that the new API deprecates events such as `TX_COMPLETE_DATA_PENDING` and only `confirm_send()` is used to pass over additional info/flags via the auxiliary info field.

Maybe it makes sense to let this PR also depend on an update of openthread?

Reply:
Or I could add a commit on top.
If the return value of `confirm_send()` is passed to the ot `send_pkt()` function and is properly evaluated, it should be the same behavior.

Reply:
If the change is not too large, that would be indeed better. Without an updated driver, it is not possible to test the new feature anyway.